validators/src/unit-test/java/uk/gov/moj/sdt/validators/SubmitQueryRequestValidatorTest.java

```diff
@@ -100,16 +101,14 @@ public class SubmitQueryRequestValidatorTest extends AbstractValidatorUnitTest {
     /**
      * Error messages cache.
      */
+    @Mock
     private ICacheable errorMessagesCache;
 
-    /**
-     * Error message.
-     */
-    private IErrorMessage errorMessage;
-
     /**
      * Setup of the Validator and Domain class instance.
      */
+    @BeforeEach

```

Comment:
Don't think this annotation is needed.  The setupLocalTests() method is invoked from setUp() on AbstractSdtUnitTestBase which is annotated with @BeforeEach.  This should make it run before each test.